src/ngraph/frontend/onnx_import/utils/reduction.cpp

```diff
@@ -80,6 +81,31 @@ namespace ngraph
                     Shape{output_shape});
             }
 
+            std::shared_ptr<ngraph::Node>
+                make_ng_dyn_reduction_op(const Node& node,
+                                         const std::shared_ptr<ngraph::Node>& ng_input,
+                                         DynamicReductionFunction reduction_function)
+            {
+                auto data_shape = ng_input->get_shape();
```

Comment:
If this function have dyn(dynamic?) in name why are you using static shape?
Does dynamic != dynamic shape?

Reply:
- I renamed `DynamicReductionFunction` to `RuntimeReductionFunction` - in this context it means using nodes instead of attributes
- `make_ng_dyn_reduction_op` was renamed to `make_ng_reduction_op` (overloaded version)